forc/README.md

```diff
@@ -86,12 +86,13 @@ async fn harness() {
     // Build the contract
     let salt: [u8; 32] = rng.gen();
     let salt = Salt::from(salt);
-    let compiled = Contract::compile_sway_contract("./", salt).unwrap();
 
-    // Launch a local network and deploy the contract
-    let (client, contract_id) = Contract::launch_and_deploy(&compiled).await.unwrap();
+    let compiled = Contract::compile_sway_contract("./", salt).unwrap();
```

Comment:
On a second look, I think eventually we're going to want to remove this in favour of passing in a pre-compiled contract (or pointing to the build directory where we can expect the contract to be, i.e. `<project>/out/<profile>/<project-name>-abi.json`). Ideally we want `fuels-rs` to be agnostic of Sway, and to break the dependency cycle between the `fuels-rs` and `sway` repos. Not sure if we still want to land this in the meantime while that gets addressed in future PRs though? cc @digorithm

Reply:
This PR just reflects the updates made in https://github.com/FuelLabs/fuels-rs/pull/113, so they are necessary because the doc has to reflect the current state of things. But I agree that the SDK should become agnostic in the long term.